README.md

```diff
@@ -47,6 +48,20 @@ A simple example that doesn't use SASL relaying:
         alias: you@yourdomain.org
 ```
 
+A simple example with virtual aliases for mail forwarding that doesn't use SASL relaying:
+```yaml
+---
+- hosts: all
+  roles:
+    - postfix
+  vars:
+    - postfix_virtual_aliases:
+      - virtual: webmaster@yourdomain.com
+        alias: personal_email@gmail.com
+      - virtual: billandbob@yourdomain.com
+        alias: bill@gmail.com, bob@gmail.com
```

Comment:
Do you mean the line where it says
`alias: bill@gmail.com, bob@gmail.com`
No, you can just write them on one line separated by commas and postfix is fine with it.